blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/QueueItemImpl.java

```diff
@@ -55,6 +57,15 @@ public int getExpectedBuildNumber() {
     }
 
     @Override
+    public void delete(){
+        if(!item.hasCancelPermission()){
+            throw new ServiceException.ForbiddenException(String.format("Not authorized to stop queue: %s", getId()));
+        }
+
+        Jenkins.getInstance().getQueue().cancel(item);
```

Comment:
does this happen on behalf of the user that requested it or is it a system level operation? (I am guessing it has no concept of user?).